wdl2cwl/main.py

```diff
@@ -197,6 +197,10 @@ def get_output(expression: str, input_names: List[str]) -> str:
         if expression.replace('"', "") in input_names:
             output_value = "$(inputs." + expression + ")"
         output_value = output_value.replace('"', "")
```

Comment:
Why the `replace()` in both of these lines? We already know that `"` is not in `expression` from the ` elif '"' not in expression:`

Reply:
Yes. Not necessary. I do it out of habit :D Modified the code